tests/models/beit/test_modeling_tf_beit.py

```diff
@@ -0,0 +1,616 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Testing suite for the TensorFlow Beit model. """
+
+
+import collections.abc
+import inspect
+import unittest
+
+from datasets import load_dataset
+from packaging import version
+
+import numpy as np
+
+from transformers import BeitConfig
+from transformers.file_utils import cached_property, is_tf_available, is_vision_available
+from transformers.testing_utils import require_tf, require_vision, slow
+
+from ...test_configuration_common import ConfigTester
+from ...test_modeling_tf_common import TFModelTesterMixin, floats_tensor, ids_tensor
+
+
+if is_tf_available():
+    import tensorflow as tf
+
+    from transformers import (
+        TFBeitForImageClassification,
+        TFBeitForMaskedImageModeling,
+        TFBeitForSemanticSegmentation,
+        TFBeitModel,
```

Comment:
This class hasn't, and needs to be, defined

Reply:
Done!